about.html

```diff
@@ -4,13 +4,28 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <script src="./mobileMenu.js"></script>
   <link rel="stylesheet" href="./about.css">
+  <link rel="stylesheet" href="./mobileMenu.css">
   <title>About E3 2023</title>
 </head>
 <body class="body">
 
+  <div class="mobileMenu">
+    <img class="logoMenu" src="/img/logo.png" alt="">
+    <a href="#" onclick="closeMobile()" class="btMob"><img class="buttonX" src="img/close_button.png" alt="CloseMenu"></a>
+    <div>
+      <a href="/index.html">Home</a>
+      <a href="#program">Program</a>
+      <a href="#join">Join</a>
+      <a href="#news">News</a>
+      <a href="#campaign">E3 Campaign</a>
+    </div>
+  </div>
```

Comment:
- [ ] I would like to recommend you move your top mobile menu bar inside your header in order to follow the standard and good practice. Also, this will help you to easily work on your design alignment :+1: